### ISSUE TITLE
Vue-final-modal do not work in typescript setup script 

### ISSUE THREAD

**Issue by riderx** [vue-final/vue-final-modal#173]
There are no documentation to access modal in setup witch is the main feautre of vue 3

### Version

vue-final-modal:  3.4.1
vue:  3.2.1

### OS

Mac

### Reproduction Link
```
<script setup lang="ts">
import { $vfm } from 'vue-final-modal'
// Module '"vue-final-modal"' has no exported member '$vfm'. Did you mean to use 'import $vfm from "vue-final-modal"' instead?ts(2614)
</script>
```

### What is Expected?
$vfm should be defined

### What is actually happening?
$vfm is undefined

**Comment by petegriffith** [comment]
Any traction here?

**Comment by soulcodex** [comment]
Please @hunterliu1003 ❤️ , this fix is very important to build projects for production avoiding errors from TS compiler and linter 😢

**Comment by hunterliu1003** [comment]
Hi

Sorry I'm not a Typescript user actually, and I'm still learning it. I would be very grateful if anyone can send a PR to fix this.

**Comment by Mini-ghost** [comment]
This issue is resolve (#222), Thanks every one. 💚

**Comment by hunterliu1003** [comment]
This PR is published at [vue-final-modal@3.4.4](https://www.npmjs.com/package/vue-final-modal/v/3.4.4)
